### ISSUE TITLE
Login with Token?

### ISSUE THREAD

**Issue by xDelito** [Kleinrotti/TeamCord#16]
I think a token login would be easier and more effective.

**Comment by Kleinrotti** [comment]
Do you mean login with a bot token or just copy the user token?

**Comment by Bealze** [comment]
> Do you mean login with a bot token or just copy the user token?

i was thinking a simple click to log into discord  button

**Comment by svenbledt** [comment]
Yes the Problem i have is to login because of 2 Step verification would be better with Token

**Comment by Rysanlos** [comment]
Hello, I'm looking for a Discord bot login instead of self bot, is it possible?
